docs/content/guides/2.alokai-essentials/1.alokai-next-js/4.install-sdk.md

```diff
@@ -5,82 +5,105 @@ navigation:
   icon: tabler:number-3-small
 ---
 
-# Install and Configure Alokai SDK
+# Install and Configure Alokai Context
```

Comment:
The guide part was originally written by Sergii. It contains examples on the github repository so those would also need to be updated. 
My recommendation is to overtake this part by the DX team so they could validate the docs and update examples on github all together.